packages/shared-base/src/sentry/index.ts

```diff
@@ -0,0 +1,78 @@
+import type { PersonaIdentifier, ProfileIdentifier } from '../Identifier/index.js'
+
+// @ts-ignore
+const Sentry = globalThis.Sentry as Sentry
+
+Sentry.init({
+    dsn: '',
+    defaultIntegrations: false,
+    integrations: [new Sentry.Integrations.Breadcrumbs({ console: false })],
+    environment: process.env.NODE_ENV,
+    release: process.env.VERSION,
+    tracesSampleRate: 1.0,
+})
+
+export const captureException = () => {
+    Sentry.captureException()
+}
+export const captureMessage = () => {
+    Sentry.captureMessage()
+}
+
+export interface LogHubBase {
+    captureException(error: Error): void
+    captureMessage(message: string | object): void
+}
+
+export class LogHub implements LogHubBase {
+    private _platform: string
+    private _plugin_id?: string
+    private _user?: {
+        persona?: PersonaIdentifier
+        profile?: ProfileIdentifier
+    }
+
+    constructor(
+        platform: string,
+        pluginId?: string,
+        user?: { persona?: PersonaIdentifier; profile?: ProfileIdentifier },
```

Comment:
Do we need this information in the logging?

Reply:
Don't need. I will remove it.